src/components/reusuables/cards/ImageCard.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
+import { StaticImage } from "gatsby-plugin-image"
+
+const ImageCard = () => {
+  return (
+    <StaticImage
```

Comment:
Can this be passed as a prop? Import it into the home.js and passed. May need GatsbyImage instead of StaticImage.

Reply:
Side note: I couldn't get it to work with GatsybyImage so I had to use bootstrap image component.

Reply:
Image card logic has been updated.